homeassistant/components/matter/climate.py

```diff
@@ -118,24 +143,32 @@ def target_temperature(self) -> float | None:
     @property
     def target_temperature_high(self) -> float | None:
         """Return the highbound target temperature we try to reach."""
-        return self._get_temperature_in_degrees(
-            clusters.Thermostat.Attributes.OccupiedCoolingSetpoint
-        )
+        match self._attr_hvac_mode:
```

Comment:
A simple `if` block is cleaner in this case. A match block is usable when there are more than a couple of cases.